v1/test/e2e/metrics/metrics_test.go

```diff
@@ -212,7 +212,7 @@ func assertDataInstrumentationMetricsInMap(t *testing.T, includeCompile bool, me
 		"timer_server_handler_ns",
 	}
 	compileStageKeys := []string{
-		"timer_rego_query_parse_ns",
+		//"timer_rego_query_parse_ns",
```

Comment:
Should we be adding time spent in `stringPathToDataRef()` as `timer_rego_query_parse_ns`?

Reply:
Sounds good.